configs/pipelines/detector_remax_convnext_novelty_detection.pipe

```diff
@@ -0,0 +1,139 @@
+
+config _pipeline:_edge
+  :capacity                                    1
+
+config _scheduler
+  :type                                        pythread_per_process
+
+# ==============================================================================
+
+process input
+  :: video_input
+  :video_filename                              input_list.txt
+  :frame_time                                  1
+  :exit_on_invalid                             false
+
+  :video_reader:type                           image_list
+
+  block video_reader:vidl_ffmpeg
+    :time_source                               start_at_0
+  endblock
+
+  block video_reader:image_list
+    :image_reader:type                         vxl
+    :skip_bad_images                           true
+
+    block image_reader:vxl
+      :force_byte                              true
+    endblock
+
+    block image_reader:add_timestamp_from_filename
+      :image_reader:type                       vxl
+
+      block image_reader:vxl
+        :force_byte                            true
+      endblock
+    endblock
+  endblock
+
+process downsampler
+  :: downsample
+  :target_frame_rate                           5
+  :burst_frame_count                           0
+  :burst_frame_break                           0
+  :renumber_frames                             true
+
+connect from input.image
+        to   downsampler.input_1
+connect from input.file_name
+        to   downsampler.input_2
+connect from input.frame_rate
+        to   downsampler.frame_rate
+connect from input.timestamp
+        to   downsampler.timestamp
+
+process detector_input
+  :: image_filter
+  :filter:type                                 vxl_convert_image
+
+  block filter:vxl_convert_image
+    :format                                    byte
+    :force_three_channel                       true
+  endblock
+
+connect from downsampler.output_1
+        to   detector_input.image
+
+# ==============================================================================
+
+process detector1
+  :: image_object_detector
+  :detector:type                               ocv_windowed
+
+  block detector:ocv_windowed
+    :detector:type                             detector_remax_convnext
+
+    :mode                                      adaptive
+    :chip_adaptive_thresh                      2000000
+    :scale                                     1.25
+    :chip_width                                1333
+    :chip_height                               800
+    :chip_step_height                          1100
+    :chip_step_width                           600
+    :chip_edge_filter                          -10
+    :batch_size                                1
+
+    block detector:detector_remax_convnext
+      # Network config, weights, and names
+      relativepath net_config  =               ../../../../src/packages/learn/learn/algorithms/MMDET/configs/convnext_extra_large_config.py
```

Comment:
Fix paths

Reply:
../../../../src/packages/learn/learn/algorithms/MMDET/configs/convnext_extra_large_config.py is not a valid config path. When running from installers, there is no source directory. All models should be in the models directory underneath these files